pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -83,3 +70,33 @@ func domainInterfaceFromNetwork(domain *api.Domain, network v1.Network) *api.Int
 	}
 	return nil
 }
+
+func networksToHotplugWhoseInterfacesAreNotInTheDomain(vmi *v1.VirtualMachineInstance, indexedDomainIfaces map[string]api.Interface) []v1.Network {
+	var networksToHotplug []v1.Network
+	interfacesToHoplug := netvmispec.IndexInterfacesFromStatus(
+		vmi.Status.Interfaces,
+		func(ifaceStatus v1.VirtualMachineInstanceNetworkInterface) bool {
+			_, exists := indexedDomainIfaces[ifaceStatus.Name]
+
+			return netvmispec.ContainsInfoSource(
+				ifaceStatus.InfoSource, netvmispec.InfoSourceMultusStatus,
+			) && !exists
+		},
+	)
+
+	for netName, network := range netvmispec.IndexNetworkSpecByName(vmi.Spec.Networks) {
+		if _, isAttachmentToBeHotplugged := interfacesToHoplug[netName]; isAttachmentToBeHotplugged {
+			networksToHotplug = append(networksToHotplug, network)
+		}
+	}
+
+	return networksToHotplug
+}
+
+func indexedDomainInterfaces(domain *api.Domain) map[string]api.Interface {
+	interfacesInDomainStatus := map[string]api.Interface{}
```

Comment:
This variable name specifies _status_ but it actually reflects the domain spec, can you please omit the word _status_ or rename it to something like `domainInterfaces`?